src/careamics_restoration/bioimage/io.py

```diff
@@ -0,0 +1,102 @@
+from pathlib import Path
+from typing import Union
+
+import yaml
+from bioimageio.core import load_resource_description
+from bioimageio.core.build_spec import build_model
+from bioimageio.spec.model.raw_nodes import Model
+
+from careamics_restoration.config.config import (
+    Configuration,
+)
+
+PYTORCH_STATE_DICT = "pytorch_state_dict"
+CAREAMICS_CONFIG = "careamics_config.yml"
+
+
+def _load_model_rdf(name: str) -> dict:
+    """Load an rdf(yaml) file given a model name."""
+    name = name.lower()
+    rdf_file = Path(__file__).parent.joinpath("rdfs").joinpath(name + ".yaml")
+    if not rdf_file.exists:
+        # a warning maybe?
+        return {}
+
+    with open(rdf_file) as f:
+        rdf = yaml.safe_load(f)
+
+    # add covers if available
+    cover_folder = Path(__file__).parent.joinpath("covers")
+    cover_images = list(cover_folder.glob(f"{name}_cover*.*"))
+    if len(cover_images) > 0:
+        rdf["covers"] = [str(img.absolute()) for img in cover_images]
+
+    # add documentation (md file)
+    doc = Path(__file__).parent.joinpath("docs").joinpath(f"{name}.md")
+    if doc.exists() > 0:
+        rdf["documentation"] = str(doc.absolute())
+
+    return rdf
+
+
+def get_default_model_specs(name: str) -> dict:
```

Comment:
I am not a fan of loading a default rdf and modifying it, because the default has then a lot of details that are potentially wrong and might find their way into the exported rdf.

Also, why bother loading something that we can have as a dict returned by a function?

So I would just pack the really important information into a dict returned by a function `get_default_n2v_rdf`

Reply:
The current `n2v.yaml` inside the `rdfs` folder contains almost all required fields. The `documentation` field should point to a `.md` file that I put it in the `docs` folder (so later we can have separate docs for different models). Also for the `covers` there is a folder.  
So these fields added to the default model's rdf dict, after loading the related yaml file.  
We can hard-code all of these somewhere, but I think having a long dict of nested dicts/lists also is not that clean...but if it is just for now and for "n2v", it could be an option.

Reply:
All the info inside the `n2v.yaml` file are required, plus the `documentation` that after loading the `yaml` file gets added to the model's rdf dict.  
We can hard-code them all, but having a long dict of nested dicts/lists is not very clean too, and I think it is not actual functioning code but some meta-data filling code.  
Anyway, It can be done that way, specially now, that we only have one model.